lib/ansible/modules/monitoring/circonus_annotation.py

```diff
@@ -91,7 +89,11 @@
 import json
 import time
 
-import requests
+try:
+    import requests
+    HAS_REQUESTS = True
```

Comment:
requests is a 'fun' library, the API has changed over time, so you should always check for a specific version (I know this is an issue with the original module, but that should have been caught before merging it.)